src/current/cockroachcloud/quickstart-trial-cluster.md

```diff
@@ -21,7 +21,7 @@ For this tutorial, you will create a 3-node GCP cluster in the `us-west2` region
 1. For **Regions & nodes**, select `California (us-west)` region and 3 nodes.
 
     {{site.data.alerts.callout_info}}
-    You can also select 3 regions with 3 nodes per region if you want to create a [multi-region]({% link cockroachcloud/plan-your-cluster.md %}#multi-region-clusters) trial cluster.
+    You can also select 3 regions with 3 nodes per region if you want to create a [multi-region]({% link cockroachcloud/plan-your-cluster-advanced.md %}#multi-region-clusters) trial cluster.
```

Comment:
```suggestion
    To create a [multi-region]({% link cockroachcloud/plan-your-cluster-advanced.md %}#multi-region-clusters) trial cluster, you can 3 regions with 3 nodes per region, 9 nodes in total.
```

Reply:
@mdlinville Grammar issue - can you fix?

Reply:
Done. will apply.